packages/ckeditor5-table/tests/tableui.js

```diff
@@ -382,6 +382,15 @@ describe( 'TableUI', () => {
 			expect( button.icon ).to.match( /<svg / );
 		} );
 
+		it( 'have button arrow with proper properties set', () => {
+			expect( dropdown ).to.be.instanceOf( DropdownView );
+
+			const buttonArrow = dropdown.buttonView.arrowView;
+
+			expect( buttonArrow.tooltip ).to.be.true;
+			expect( buttonArrow.label ).to.equal( 'Merge cells' );
+		} );
+
```

Comment:
Obsolete per my comment in [codeblockui.js](https://github.com/ckeditor/ckeditor5/pull/11859/files?show-viewed-files=true&file-filters%5B%5D=#diff-a75a67319b38c5dc7c27b30eeaff970892cdd73e01b28920a569d161458ea8c7).